docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -196,10 +203,49 @@ The following is the runtime configuration to specify:
+
+To map LDAP groups to YugabyteDB Anywhere roles, do the following:
+
+1. Navigate to **Admin > User Management > User Authentication** and select **LDAP Configuration**.
+
+1. Under **Role Settings**, enable the **Define Role to Group Mapping** option.
+
+1. Choose how to look up LDAP group membership:
+
+    - Select **User Attribute** and set the name of the LDAP user attribute to use to find the groups that users belong to.
+    - Select **Group Search Filter** and enter an LDAP search filter to search for membership in group member listings. To specify a YugabyteDB Anywhere user in the filter, use the string `{username}`. For all occurrences of this string in the query filter, YBA will replace those with the actual username used to log in to YugabyteDB Anywhere. Enter a group search base DN to use for the group search. Use the scope option to set the scope of the search; there are three levels - SUBTREE, ONELEVEL, and OBJECT.
+
+1. Click **Create Mappings** (or, if you have existing mappings, **Edit**) to display the **Create Mapping** dialog.
+
+1. Click **Add rows** to add mappings. Select the YugabyteDB Anywhere role and enter the LDAP Group DN that you want to map the role to.
 
-You need to define a YugabyteDB Anywhere-specific role for each user on your LDAP server by setting the `yugabytePlatformRole` annotation. The value set for this annotation is read during the YugabyteDB Anywhere login. Note that if the value is modified on the LDAP server, the change is propagated to YugabyteDB Anywhere and automatically updated during login. Password updates are also automatically handled.
+1. Click **Confirm**.
+
+1. Click **Save** when you are done.
+
+### Define the YugabyteDB Anywhere role
+
+You need to define a YugabyteDB Anywhere-specific role for each user on your LDAP server by setting the `yugabytePlatformRole` annotation on the LDAP server. The value set for this annotation is read during the YugabyteDB Anywhere login. Note that if the value is modified on the LDAP server, the change is propagated to YugabyteDB Anywhere and automatically updated during login. Password updates are also automatically handled.
 
 If the role is not specified, users are created with ReadOnly privileges by default, which can be modified by the local super admin.
 
 When LDAP is set up on a Windows Active Directory (AD) server, the user is expected to have permissions to query the user's properties from that server. If the permissions have not been granted, YugabyteDB Anywhere defaults its role to ReadOnly, which can later be modified by the local super admin.
+
+When a LDAP user logs in to YugabyteDB Anywhere, the system handles role assignment as follows:
+
+- If group mapping is enabled:
+
+  - If YugabyteDB Anywhere can obtain a valid role from the LDAP server (via `yugabytePlatformRole` and group mappings), the system assigns the role to the user. Otherwise, the user is assigned the ReadOnly role.
```

Comment:
We need to explain that the final valid role that we obtain, is a combination of the role we get from `yugabytePlatformRole` and any roles that are mapped to the groups of which the logged in user is a member. 

Meaning, it is possible we get nothing from `yugabytePlatformRole` or that there is no mapped group membership. We just need atleast `yugabytePlatformRole` to be defined for the user in the LDAP server or we need at least one such group which is mapped to a role in YBA and the user who is logging in is also a member of it. Only if both of these conditions are unfulfilled do we say that we could not obtain a valid role from the LDAP server.

This will remain the same in the newer versions as well.